lib/core/src/server/config/utils.js

```diff
@@ -25,7 +25,16 @@ const projectRoot = () => {
 
 export const includePaths = [projectRoot()];
 export const nodeModulesPaths = path.resolve('./node_modules');
-export const excludePaths = [/node_modules/];
+
+// Run `npx are-you-es5 check lib/core -r`
```

Comment:
How long does it take to run this command, could we potentially run it before running storybook on the user's machine?

Reply:
It's pretty quick but it's only CLI and the output is as follow


$ npx are-you-es5 check lib/core -r
npx : 4 installé(s) en 2.09s
❌ @storybook/node-logger is not ES5
⚠️ @types/node-fetch was not checked because no entry script
 was found
❌ better-opn is not ES5
❌ boxen is not ES5
❌ chalk is not ES5
❌ commander is not ES5
❌ find-cache-dir is not ES5
❌ find-up is not ES5
❌ fs-extra is not ES5
❌ json5 is not ES5
❌ node-fetch is not ES5
❌ pkg-dir is not ES5
⚠️ react-dev-utils was not checked because no entry script w
as found
❌ resolve-from is not ES5
❌ semver is not ES5


Babel-loader exclude regex:
/[\\/]node_modules[\\/](?!(@storybook\/node-logger|better-opn|boxen|chalk|commander|find-cache-dir|find-up|fs-extra|json5|node-fetch|pkg-dir|resolve-from|semver)[\\/])/

Reply:
if it's expensive, we should make it easier to update this code using a script?